From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-memfd-add-mfd_noexec_seal-and-mfd_exec-fix-fix
Date: Fri Dec 16 01:31:26 PM PST 2022

remove unneeded register_pid_ns_ctl_table_vm() stub, per Jeff

Cc: Jeff Xu <jeffxu@google.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---


--- a/kernel/pid_sysctl.h~mm-memfd-add-mfd_noexec_seal-and-mfd_exec-fix-fix
+++ a/kernel/pid_sysctl.h
@@ -54,7 +54,6 @@ static inline void register_pid_ns_sysct
 #else
 static inline void initialize_memfd_noexec_scope(struct pid_namespace *ns) {}
 static inline void set_memfd_noexec_scope(struct pid_namespace *ns) {}
-static inline void register_pid_ns_ctl_table_vm(void) {}
 static inline void register_pid_ns_sysctl_table_vm(void) {}
 #endif
 
_
